### PR TITLE
[OSDOCS-8215]: 4.14z Updating hosted control planes TP table, adding new features & enhancements

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.14+

Issue: https://issues.redhat.com/browse/OSDOCS-8215

Link to docs preview: https://66219--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-technology-preview (scroll to "Hosted control plane Technology Preview features")

QE review:
- [x] QE has approved this change.

Additional information: 
This PR is for the ACM 2.9/MCE 2.4 release that is scheduled for Nov. 20. Do not merge until this date.
The links to the RHACM docs will not work until Nov. 20, when the docs go live.

This PR has received approvals from stakeholders per the OCP docs change management process.

### CODE REVIEW

**Review by zanetworker — APPROVED** [review]

**Comment by YuLi517** [comment]
Please add known issue:  OCPBUGS-22753. Description can be same as https://github.com/stolostron/rhacm-docs/pull/5777.

```
[#hosted-cluster-dual-stack]
=== Potential DNS issues in hosted clusters configured for a dual-stack, disconnected environment

When you create a hosted cluster in a disconnected environment that uses the dual-stack network, you might encounter the following DNS-related issues:

* `CrashLoopBackOff` state in the `service-ca-operator` pod: When the pod tries to reach the Kubernetes API server through the hosted control plane, the pod cannot reach the server because the data plane proxy in the `kube-system` namespace cannot resolve the request. This issue occurs because in the HAProxy setup, the front end uses an IP address and the back end uses a DNS name that the pod cannot resolve.
* Pods stuck in `ContainerCreating` state: This issue occurs because the `openshift-service-ca-operator` cannot generate the `metrics-tls` secret that the DNS pods need for DNS resolution. As a result, the pods cannot resolve the Kubernetes API server.

To resolve those issues, configure the DNS server settings by following the guidelines in xref:../hosted_control_planes/dual_stack_dns.adoc#dual-stack-dns[Configuring DNS for a dual stack network].
```

**Comment by lahinson** [comment]
@YuLi517 I added the known issue as you requested. PTAL when you can and let me know if anything should change. Thanks!

**Comment by xltian** [comment]
lgtm

**Comment by YuLi517** [comment]
As QE found dual stack DNS issue also exist on connected env. I suggest to remove **disconnected environment** in the description.  **Known issue:**  OCPBUGS-22753(disconnected env) | OCPBUGS-23234 (connected env)
```
[#hosted-cluster-dual-stack]
=== Potential DNS issues in hosted clusters configured for a dual-stack

When you create a hosted cluster by using the dual-stack network, you might encounter the following DNS-related issues:

* `CrashLoopBackOff` state in the `service-ca-operator` pod: When the pod tries to reach the Kubernetes API server through the hosted control plane, the pod cannot reach the server because the data plane proxy in the `kube-system` namespace cannot resolve the request. This issue occurs because in the HAProxy setup, the front end uses an IP address and the back end uses a DNS name that the pod cannot resolve.
* Pods stuck in `ContainerCreating` state: This issue occurs because the `openshift-service-ca-operator` cannot generate the `metrics-tls` secret that the DNS pods need for DNS resolution. As a result, the pods cannot resolve the Kubernetes API server.

To resolve those issues, configure the DNS server settings by following the guidelines in xref:../hosted_control_planes/dual_stack_dns.adoc#dual-stack-dns[Configuring DNS for a dual stack network].
```

**Comment by lahinson** [comment]
@YuLi517 Thank you for the update. I revised the known issue in this PR to no longer mention disconnected and to also reference OCPBUGS-23234. The RHACM docs are currently frozen, but I will update the related known issue in those docs as soon as I'm able to.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1276,10 +1276,21 @@ With this release, you can remove the partitioning table before installation by
...
+Hosted control planes for {product-title} is now Generally Available on the bare-metal and {VirtProductName} platforms. Hosted control planes on AWS remains a Technology Preview feature.
+
+[id="ocp-4-14-hcp-arm"]
+==== Creating ARM NodePool objects on AWS hosted clusters (Technology Preview)
+In this release, you can schedule application workloads on 64-bit ARM and AMD64 from the same hosted control plane. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#hosted-cluster-arm-node-pools[Creating ARM NodePool objects on AWS hosted clusters].
+
+[id="ocp-4-14-hcp-ibm-power-z"]
+==== Hosted control planes on {ibmpowerProductName} and {ibmzProductName} (Technology Preview)
+In this release, hosted control planes is available as a Technology Preview feature on {ibmpowerProductName} and {ibmzProductName}.
+//Add links to IBM Power and IBM Z docs when available
```
Comment (skrthomas) [inline release_notes/ocp-4-14-release-notes.adoc:1293]: Just a note, still need links here.

**Review by skrthomas — APPROVED** [review]
One small note, lgtm otherwise

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (lahinson) [reply release_notes/ocp-4-14-release-notes.adoc:1293]: Good catch! I'll add those.

**Comment by lahinson** [comment]
/label merge-review-needed

Note to reviewers: This PR should not be merged until Nov. 20, in alignment with the ACM 2.9 GA. I'm requesting a merge review in advance per the new process: https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#heading=h.x8ydklaezdcq

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -2679,15 +2689,25 @@ In the following tables, features are marked with the following statuses:
...
-|Technology Preview
-// Needs to move to GA after Nov 15, 2023
+|General Availability
+
 |Hosted control planes for {product-title} on {VirtProductName}
 |Not Available
 |Technology Preview
+|General Availability
+
+|Hosted control planes for {product-title} on {ibmpowerProductName}
```
Comment (bburt-rh) [inline release_notes/ocp-4-14-release-notes.adoc:2703]: This attribute is not rendering in the preview. Is it being defined in a different PR that hasn't merged yet?

image

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -2679,15 +2689,25 @@ In the following tables, features are marked with the following statuses:
...
+|Not Available
+|Not Available
+|Technology Preview
+
+|Hosted control planes for {product-title} on {ibmzProductName}
```
Comment (bburt-rh) [inline release_notes/ocp-4-14-release-notes.adoc:2708]: This attribute also is not rendering in the preview.

**Review by bburt-rh — COMMENTED** [review]
Not sure what's going with the attributes in the TP table not rendering, but the issue needs to be resolved/explained before I can set this PR as `good-to-merge`. Thanks, @lahinson!

File: release_notes/ocp-4-14-release-notes.adoc (hunk 3 above)
Comment (lahinson) [reply release_notes/ocp-4-14-release-notes.adoc:2708]: I think the attributes file was recently updated. I'll double-check and replace the ibmzProductName attribute with whatever the new attribute is. Thanks!

Comment (lahinson) [reply release_notes/ocp-4-14-release-notes.adoc]: @bburt-rh The attributes are fixed now. Thanks again!

Comment (bburt-rh) [reply release_notes/ocp-4-14-release-notes.adoc]: Great. Thanks, @lahinson! LGTM now. Giving it the good-to-merge label

**Pull request merged**